ppr-ui/src/composables/mhrInformation/useTransferOwners.ts

```diff
@@ -50,6 +50,13 @@ export const useTransferOwners = (enableAllActions: boolean = false) => {
     ].includes(getMhrTransferType.value?.transferType)
   })
 
+  /** Returns true when Add/Edit Owner name fields should be disabled **/
+  const disableNameFields = computed((): boolean => {
+    return [
+      ApiTransferTypes.SURVIVING_JOINT_TENANT
+    ].includes(getMhrTransferType.value?.transferType)
+  })
+
```

Comment:
Good addition to get more granular control (ie replacing just the 'due to death' computed value.

Reply:
can also add onto this depending on the other flows

Reply:
Absolutely!